.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behaviour
+
+### Actual Behaviour
+
+### Reproduce Scenario (including but not limited to)
```

Comment:
What's the different between "Reproduce scenario" and "Steps to reproduce" below?

Reply:
I think one is summary one are the exact details. Templates were provided by open source guys.